okta/AppInstanceClient.py

```diff
@@ -131,17 +135,6 @@ def deactivate_app_instance(self, id):
 
     # USER
 
-    def get_app_assignments_for_user(self, user, expand_user=True):
```

Comment:
Why remove `get_app_assignments_for_user` rather than rewriting it as a wrapper over `get_all_instances`?

Reply:
Actually `get_all_instances` doesn't exist in okta-sdk. It's implemented in CR, so this snippet is mostly from CloudRunner. So, I decided to make CR and okta-sdk code consistent and do not create extra methods here. All necessary wrappers will be implemented in CR:

Here is how it looks in CR in `api.utils.app`:

```
def get_all_user_apps(user_id, expand_user=True):
    expand = f"user/{user_id}" if expand_user else None
    filter_string = f"user.id eq \"{user_id}\""
    return utils.get_all_instances('app', 'get_paged_app_instances',
                                   filter_string=filter_string, expand=expand)
```

And here how `get_all_instances` is implemented in `api.utils.base`

```
def get_all_instances(model_name, client_method_name, *args, **kwargs):
    client = get_okta_client(model_name)
    is_done = False
    next_url = None
    instances = {}
    while not is_done:
        paged_result = getattr(client, client_method_name)(
            *args, limit=200, url=next_url, **kwargs)
        for instance in paged_result.result:
            instances[instance.id] = instance
        if paged_result.is_last_page():
            is_done = True
        else:
            next_url = paged_result.next_url
    return instances
```

Reply:
Alright, now I get it. Sorry for the confusion.